src/layer.rs

```diff
@@ -11,6 +15,17 @@ pub enum LayerType {
     /// A layer that groups other layers and does not contain any image data.
     /// In Aseprite these are represented by a folder icon.
     Group,
+    /// A tilemap layer. Contains the tileset index.
+    /// In Aseprite these are represented by a grid icon.
+    Tilemap(TilesetId),
+}
+impl LayerType {
+    pub(crate) fn expect_tilemap(&self) -> &TilesetId {
+        match self {
+            Self::Tilemap(id) => id,
+            _ => panic!("Expected a Tilemap layer"),
```

Comment:
Another panic. Should either be turn into a `Result` or made impossible by validating first.